bedrock/firefox/templates/firefox/dev-firstrun.html

```diff
@@ -147,7 +137,31 @@
             <div class="video" data-video-id="R_qDaLQ8ghg"></div>
           </div>
         </li>
-      {% endif %}
+        <li class="feature">
+          <a href="https://developer.mozilla.org/docs/Tools/Performance" rel="external">
+            <img src="{{ static('img/firefox/firstrun/dev/performance-tools.png') }}" alt="{{ _('Screenshot') }}" class="screenshot">
+          </a>
+          <h2>{{_('Performance Tools')}}</h2>
```

Comment:
This one is now duplicated but, I do not think we can get around it unless, it is ok to have only 1 featured item at the top of the page for locales that does not have `firefox_dev_fall` enabled